test/common/util.ts

```diff
@@ -57,9 +45,6 @@ export async function getElementsText(
  * Once that happens, pull this out of `util.ts` and move it someplace more page-object-y.
  */
 export var selectors = {
-    nav: "#stage > #nav",
-    components: "#stage",
```

Comment:
Did you remove `components` and `nav` on purpose?

Reply:
Not on purpose, it was not being used anywhere in our code. Removed it as part of cleaning the code.